assets/src/containers/FilesAccessed.js

```diff
@@ -56,6 +60,16 @@ function FilesAccessed (props) {
   const [defaultCheckboxState, setDefaultCheckedState] = useState(true)
   const [defaultLabel, setDefaultLabel] = useState(currentSetting)
 
+  function setDefaultFilterState() {
```

Comment:
Could you explain what this function is doing? The `set` prefix suggests setting some state, but I don't think it is setting any state. It may be worth renaming. 

I also think it can be refactored to be shorter and more readable:

```js
function setDefaultFilterState(resourceValues) {
   return resourceValues.filter(resourceItem => resourceItem.disabled === 'false')
}
```

Reply:
Oh, this function creates an array that will be used as the default state of the `resourceFilter` state. Thanks for the suggestions.